grakn-graph/src/main/java/ai/grakn/graph/internal/ConceptLog.java

```diff
@@ -216,4 +227,16 @@ boolean isTypeCached(TypeLabel label){
         //noinspection unchecked
         return (X) typeCache.get(label);
     }
+
+    void addedInstance(TypeLabel name){
+        instanceCount.compute(name, (key, value) -> value == null ? 1 : value + 1);
```

Comment:
Same question. What if the value does not exist? Does it just put the new value in?